test/adapter/polymorphic_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
```

Comment:
Yeah, the tests are currently a bloody mess. We really need to do something about it.